sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -1403,6 +1430,46 @@ public virtual void UpdateSyncToken(string token)
             _syncTokenPolicy.AddToken(token);
         }
 
+        /// <summary>
+        /// Parses the response of a <see cref="GetConfigurationSettings(SettingSelector, CancellationToken)"/> request.
```

Comment:
Wow.   That's an odd experience.   Do they always return the "Link" header or only for a 304?

Reply:
In the responses I checked they seemed to return it every time, but I have no explicit confirmation from the service so I decided to keep both approaches just in case.